### ISSUE TITLE
installing version 1.0.1 fails with error

### ISSUE THREAD

**Issue by jerryfoutz** [gruntjs/grunt-contrib-connect#213]
Installing version 1.0.0 **npm install grunt-contrib-connect@1.0.0** or **npm install grunt-contrib-connect --save-dev** both work, but it fails  with **npm install grunt-contrib-connect@1.0.1**

Failing install command below and log attached.

> npm install grunt-contrib-connect@1.0.1
npm WARN addRemoteGit Error: Command failed: git -c core.longpaths=true config -
-get remote.origin.url
npm WARN addRemoteGit
npm WARN addRemoteGit     at ChildProcess.exithandler (child_process.js:202:12)
npm WARN addRemoteGit     at emitTwo (events.js:100:13)
npm WARN addRemoteGit     at ChildProcess.emit (events.js:185:7)
npm WARN addRemoteGit     at maybeClose (internal/child_process.js:850:16)
npm WARN addRemoteGit     at Process.ChildProcess._handle.onexit (internal/child
_process.js:215:5)
npm WARN addRemoteGit  git+https://github.com/gruntjs/node-http2.git#fix-return-
value resetting remote C:\Users\[user_name]\AppData\Roaming\npm-cache\_git-remo
tes\git-https-github-com-gruntjs-node-http2-git-fix-return-value-b524b70c becaus
e of error: { [Error: Command failed: git -c core.longpaths=true config --get re
mote.origin.url
npm WARN addRemoteGit ]
npm WARN addRemoteGit   killed: false,
npm WARN addRemoteGit   code: 1,
npm WARN addRemoteGit   signal: null,
npm WARN addRemoteGit   cmd: 'git -c core.longpaths=true config --get remote.ori
gin.url' }
npm ERR! git clone --template=C:\Users\[user_name]\AppData\Roaming\npm-cache\_g
it-remotes\_templates --mirror https://github.com/gruntjs/node-http2.git C:\User
s\[user_name]\AppData\Roaming\npm-cache\_git-remotes\git-https-github-com-grunt
js-node-http2-git-fix-return-value-b524b70c: Cloning into bare repository 'C:\Us
ers\[user_name]\AppData\Roaming\npm-cache\_git-remotes\git-https-github-com-gru
ntjs-node-http2-git-fix-return-value-b524b70c'...
npm ERR! git clone --template=C:\Users\[user_name]\AppData\Roaming\npm-cache\_g
it-remotes\_templates --mirror https://github.com/gruntjs/node-http2.git C:\User
s\[user_name]\AppData\Roaming\npm-cache\_git-remotes\git-https-github-com-grunt
js-node-http2-git-fix-return-value-b524b70c: fatal: unable to connect to github.
com:
npm ERR! git clone --template=C:\Users\[user_name]\AppData\Roaming\npm-cache\_g
it-remotes\_templates --mirror https://github.com/gruntjs/node-http2.git C:\User
s\[user_name]\AppData\Roaming\npm-cache\_git-remotes\git-https-github-com-grunt
js-node-http2-git-fix-return-value-b524b70c: github.com[0: 192.30.252.130]: errn
o=Invalid argument
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\[user_name]\\Ap
pData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install" "grunt-contri
b-connect@1.0.1"
npm ERR! node v5.9.0
npm ERR! npm  v3.8.2
npm ERR! code 128

npm ERR! Command failed: git -c core.longpaths=true clone --template=C:\Users\ge
rald.foutz\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror https://gi
thub.com/gruntjs/node-http2.git C:\Users\[user_name]\AppData\Roaming\npm-cache\
_git-remotes\git-https-github-com-gruntjs-node-http2-git-fix-return-value-b524b7
0c
npm ERR! Cloning into bare repository 'C:\Users\[user_name]\AppData\Roaming\npm
-cache\_git-remotes\git-https-github-com-gruntjs-node-http2-git-fix-return-value
-b524b70c'...
npm ERR! fatal: unable to connect to github.com:
npm ERR! github.com[0: 192.30.252.130]: errno=Invalid argument
npm ERR!
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     c:\a\npm-debug.log

[user_name]@DEWPAG28913 c:\a
> npm install grunt-contrib-connect@1.0.1
npm WARN addRemoteGit Error: Command failed: git -c core.longpaths=true config --get remote.origin.url
npm WARN addRemoteGit
npm WARN addRemoteGit     at ChildProcess.exithandler (child_process.js:202:12)
npm WARN addRemoteGit     at emitTwo (events.js:100:13)
npm WARN addRemoteGit     at ChildProcess.emit (events.js:185:7)
npm WARN addRemoteGit     at maybeClose (internal/child_process.js:850:16)
npm WARN addRemoteGit     at Socket.<anonymous> (internal/child_process.js:323:11)
npm WARN addRemoteGit     at emitOne (events.js:90:13)
npm WARN addRemoteGit     at Socket.emit (events.js:182:7)
npm WARN addRemoteGit     at Pipe._onclose (net.js:479:12)
npm WARN addRemoteGit  git+https://github.com/gruntjs/node-http2.git#fix-return-value resetting remote C:\Users\[user_name]\AppData\Roaming\npm-cache\_git-remotes\git-https-github-com-gruntjs-node-http2-git-fix-return-value-b524b70c because of error: { [Error: Command failed: git -c core.longpaths=true config --get remote.origin.url
npm WARN addRemoteGit ]
npm WARN addRemoteGit   killed: false,
npm WARN addRemoteGit   code: 1,
npm WARN addRemoteGit   signal: null,
npm WARN addRemoteGit   cmd: 'git -c core.longpaths=true config --get remote.origin.url' }
npm ERR! git clone --template=C:\Users\[user_name]\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror https://github.com/gruntjs/node-http2.git C:\Users\[user_name]\AppData\Roaming\npm-cache\_git-remotes\git-https-github-com-gruntjs-node-http2-git-fix-return-value-b524b70c: Cloning into bare repository 'C:\Users\[user_name]\AppData\Roaming\npm-cache\_git-remotes\git-https-github-com-gruntjs-node-http2-git-fix-return-value-b524b70c'...
npm ERR! git clone --template=C:\Users\[user_name]\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror https://github.com/gruntjs/node-http2.git C:\Users\[user_name]\AppData\Roaming\npm-cache\_git-remotes\git-https-github-com-gruntjs-node-http2-git-fix-return-value-b524b70c: fatal: unable to connect to github.com:
npm ERR! git clone --template=C:\Users\[user_name]\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror https://github.com/gruntjs/node-http2.git C:\Users\[user_name]\AppData\Roaming\npm-cache\_git-remotes\git-https-github-com-gruntjs-node-http2-git-fix-return-value-b524b70c: github.com[0: 192.30.252.131]: errno=Invalid argument
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\[user_name]\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install" "grunt-contrib-connect@1.0.1"
npm ERR! node v5.9.0
npm ERR! npm  v3.8.2
npm ERR! code 128

npm ERR! Command failed: git -c core.longpaths=true clone --template=C:\Users\[user_name]\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror https://github.com/gruntjs/node-http2.git C:\Users\[user_name]\AppData\Roaming\npm-cache\_git-remotes\git-https-github-com-gruntjs-node-http2-git-fix-return-value-b524b70c
npm ERR! Cloning into bare repository 'C:\Users\[user_name]\AppData\Roaming\npm-cache\_git-remotes\git-https-github-com-gruntjs-node-http2-git-fix-return-value-b524b70c'...
npm ERR! fatal: unable to connect to github.com:
npm ERR! github.com[0: 192.30.252.131]: errno=Invalid argument
npm ERR!
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     c:\a\npm-debug.log
[npm-debug.log.txt](https://github.com/gruntjs/grunt-contrib-connect/files/187577/npm-debug.log.txt)

**Comment by jrehwaldt** [comment]
Same on Linux with old Node:
```
npm ERR! not a package /tmp/npm-3831-DUGsxiu7/github.com/gruntjs/node-http2
...
npm ERR! Error: ENOENT, open '/tmp/npm-3831-DUGsxiu7/github.com/gruntjs/node-http2-unpack/package.json'
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Linux 3.16.0-4-amd64
npm ERR! command "/node-10/bin/node" "/node-10/bin/npm" "install"
npm ERR! cwd xxx
npm ERR! node -v v0.10.33
npm ERR! npm -v 1.4.28
npm ERR! path /tmp/npm-3831-DUGsxiu7/github.com/gruntjs/node-http2-unpack/package.json
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! not ok code 0
```

**Comment by FagnerMartinsBrack** [comment]
Same here, see https://github.com/js-cookie/js-cookie/pull/160. We got a minor upgrade PR for `1.0.1` using greenkeeper and the build is failing. Since this is a minor bump it should not contain backwards incompatible changes, [this is the offending diff](https://github.com/gruntjs/grunt-contrib-connect/compare/c368fc40c331666975098301f3f91d77e55b25b7...cd9476b11fb4dfbcb542cc4c3d5a9af729ae4a68).

**Comment by TheSavior** [comment]
We had to drop our version to 1.0.0 as well to not be breaking on this.

**Comment by vladikoff** [comment]
Should be fixed in 1.0.2
